Twelve Days: add instructions.append and hints (#3050)
exercism/python: exercises/practice/twelve-days/.docs/instructions.append.md

# Instructions append

## Test-driven development

 Be sure to check the comments at the top of the tests file.
 They will be helpful in understanding the expected output of your function.

> This _could_ be solved with a minimum of code and a lot of hardcoded text which is duplicated between the verses.

Reviewer: ```suggestion
This challenge _could_ be solved with a minimum of code and a lot of hardcoded text which is duplicated between the verses.
```

> For more of a programming challenge, consider assembling each verse from its parts.

Reviewer: ```suggestion
However, this song is a type of [cumulative song][cumulative song], where each new verse builds on the verse that comes before it.
For more of a programming challenge, consider assembling each verse from its parts.
```

> For example, for each verse, the day changes and the gift changes, and the rest of the verse is pretty much the same between verses.

Reviewer: ```suggestion
For example, for each verse, the day changes and the gift changes, and the rest of the verse is pretty much the same.
What might be a good strategy for avoiding repetition?

[cumulative song]: https://en.wikipedia.org/wiki/Cumulative_song
```
